### ISSUE TITLE
Including other files

### ISSUE THREAD

**Issue by dhbwmawhd** [aslushnikov/latex-online#18]
The documentation states that include doesn't work. Any plans on implementing that functionality?

**Comment by aslushnikov** [comment]
You can compile the whole GIT repository: https://github.com/aslushnikov/latex-online#compile-git-repo
This will handle all the includes. Example: [my diploma](latex.aslushnikov.com/compile?git=https://github.com/aslushnikov/diplom-latex&target=diplom.tex)

**Comment by dhbwmawhd** [comment]
Doesn't work for me, compare:

* https://latex.aslushnikov.com/compile?git=https://github.com/whd14d2/vorlage&target=Thesis.tex
* https://github.com/whd14d2/vorlage/releases/latest (`output.pdf` there)

**Comment by aslushnikov** [comment]
Indeed, thank you. 

I'll investigate this when I'll have time

**Comment by dhbwmawhd** [comment]
Random first thought: Maybe related to the class file being in a subdirectory?

Anyhow, will report back here in case I found the reason.

**Comment by aslushnikov** [comment]
Does this work as expected now?

**Comment by dhbwmawhd** [comment]
Link now gives me a 500 error.

**Comment by dhbwmawhd** [comment]
Oh, disregard. I had https instead of http.

---

`makeglossaries` doesn't work as far as I can tell.

**Comment by aslushnikov** [comment]
Yeah, this seems to be a custom build step. This kind of functionality is not supported.

**Comment by dhbwmawhd** [comment]
Ok.

**Comment by dhbwmawhd** [comment]
If you find time, I'd still appreciate the ability to `\makeglossaries`. Compare:

* https://en.wikibooks.org/wiki/LaTeX/Glossary

**Comment by aslushnikov** [comment]
I've started reading about this. Is it a de-facto standard for creating glossaries in TeX?

**Comment by dhbwmawhd** [comment]
The `\makeglossaries` command is standard, yes.

The desired effect can be adjusted though, as with any latex command.
